hummingbot/market/coinbase_pro/coinbase_pro_in_flight_order.pxd

```diff
@@ -0,0 +1,5 @@
+from hummingbot.market.in_flight_order_base cimport InFlightOrderBase
+
+cdef class CoinbaseProInFlightOrder(InFlightOrderBase):
+    cdef:
```

Comment:
You can just write "pass" here without the cdef if you're not defining anything new.